www/index.js

```diff
@@ -739,10 +742,29 @@ function init() {
     show("#details_custom");
   }
 
-  setupSelectList($("#versions"), Object.keys(config.versions), (version) => {
+  let versions = await fetch(config.image_url + "/.versions.json", { cache: "no-cache" })
```

Comment:
What is the point of putting the versions from the `config.json` to a `.versions.json`. I do not understand yet. :-)

Reply:
.versions.json is automatically created be the downloads.openwrt.org server while the config requires an external service. The idea is to have firmware-selector as a website hosted that doesn't need extra scripts to run